python/orca/src/bigdl/orca/learn/tf2/ray_estimator.py

```diff
@@ -391,14 +391,12 @@ def evaluate(self, data, batch_size=32, num_steps=None, verbose=1,
                                                                   data_config)
                 remote_worker_stats.append(worker.validate.remote(**params))
             worker_stats = ray.get(remote_worker_stats)
-            worker_stats = list(itertools.chain.from_iterable(worker_stats))
```

Comment:
should not flatten the list. Previously this conversion is only for creator function and ray dataset, this is wrong and makes different inputs have different output types.

Reply:
ray.get should directly return a list of evaluation result dicts and we get the first one (since all are the same)